src/VirtoCommerce.Platform.Core/PlatformOptions.cs

```diff
@@ -41,6 +42,11 @@ public class PlatformOptions
         //server (ASOS) handles the incoming requests to arriving on non-HTTPS endpoints should be rejected or not. By default, this property is set to false to help
         //mitigate man-in-the-middle attacks.
         public bool AllowInsecureHttp { get; set; }
-    }
 
+        // Time before first registration required
+        public TimeSpan RegistrationDelay { get; set; }
```

Comment:
Maybe you could add some default values for new settings here.
And it would be good to add their description into the [docs](https://github.com/VirtoCommerce/vc-platform/blob/master/docs/user-guide/configuration-settings.md)

Reply:
Good point. Updated docs.